solidity/contracts/bridge/BridgeState.sol

```diff
@@ -495,7 +501,8 @@ library BridgeState {
     ///        timeout receives. The value must be in the range [0, 100]
     /// @dev Requirements:
     ///      - Moving funds transaction max total fee must be greater than zero
-    ///      - Moving funds dust threshold must be greater than zero
+    ///      - Moving funds dust threshold must be greater than zero and smaller
```

Comment:
```suggestion
    ///      - Moving funds dust threshold must be greater than zero and lower
```

Reply:
Done in https://github.com/keep-network/tbtc-v2/pull/241/commits/f2a5d262f26294ddcbdaf4295aa33b3cc18a5418.